go/lambda-api-ci/cmd/lambda/main.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
Why is this under `cmd`?

Reply:
This is a common go convention for CLI tools / binaries